src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -513,10 +649,22 @@ private String makeArgumentDescription(final ArgumentDefinition argumentDefiniti
         return sb.toString();
     }
 
+    private void usageForPluginDescriptorArgument(final ArgumentDefinition argDef, final StringBuilder sb) {
+        final GATKCommandLinePluginDescriptor<?> descriptor = (GATKCommandLinePluginDescriptor<?>) argDef.parent;
+        // this argument came from a plugin descriptor; delegate to get the list of allowed values
+        final Set<String> allowedValues = descriptor.getAllowedValuesForDescriptorArgument(argDef.getLongName());
+        if (allowedValues == null) {
+            sb.append("Any value allowed");
+        } else {
+            sb.append("Allowed values are: ");
```

Comment:
Sort these so they're output in alphabetical order (or if you want the plugin descriptor to control the order sort them there.